invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsField.js

```diff
@@ -27,10 +27,10 @@ const creatibutorNameDisplay = (value) => {
 
   const familyName = _get(value, "person_or_org.family_name", "");
   const givenName = _get(value, "person_or_org.given_name", "");
-  const affiliationName = _get(value, `affiliations[0].name`, "");
+  const affiliations = value?.affiliations.map((affiliation) => affiliation.name);
```

Comment:
minor: You can use the https://lodash.com/docs/4.17.15#map
```suggestion
  const affiliations = _map(value?.affiliations, "name")
```

Reply:
thanks! changed